pkg/integrations/v4/logs/cfg.go

```diff
@@ -26,6 +26,7 @@ const (
 	stagingEndpoint         = "https://staging-log-api.newrelic.com/log/v1"
 	logRecordModifierSource = "nri-agent"
 	defaultBufferMaxSize    = 128
+	memBufferLimit          = 16384
```

Comment:
shouldn't this value be the default, in case memBufferLimit is not specified in the yaml config file?

Reply:
No, that value is not provided as parameter in the config file and the Logging team has decided to make this as an internal constant.